app/models/split.rb

```diff
@@ -6,6 +6,13 @@ class Split < ActiveRecord::Base
   has_many :bulk_assignments, dependent: :nullify
   has_many :variant_details, dependent: :nullify
 
+  has_many :remote_kills, ->(split) { where(split: split) }, through: :owner_app, class_name: 'AppRemoteKill', inverse_of: :split
+  has_many :feature_completions,
+    ->(split) { where(feature_gate: split) },
```

Comment:
same here, but in this case the foreign key has a non-conventional name: `feature_gate_id`

Reply:
Feature completions are per app, I believe remote kills are as well but I have to check on that. 

So in practice the feature could be complete for one app and not another.

Reply:
yeah, that makes sense, but i think that means we shouldn't use a `through`. both split and feature completions `belong_to` an `app`, if in practice, those are always the same app, then using a through shouldn't make a difference, it'll just be an extra unnecessary join. if they can be different, then the `through` will exclude some feature completions from different apps.

i guess it hinges on that question. i could imagine that you can have a feature completion on a split that an app doesn't own, but tbh, i have no idea.

Reply:
An app def can have a feature completion on a split owned by another app. Remote kills are also per app and can be against a split owned by any app.

Reply:
right! i'm limiting this too much. good call. i'm gonna make it a regular association so we properly show all the completions and remote kills across all apps for the split